CHANGELOG.md

```diff
@@ -28,6 +28,13 @@ Misc:
 
 ### [v0.10.3 (2016-11-21)](https://github.com/rails-api/active_model_serializers/compare/v0.10.2...v0.10.3)
 
+Known Issues:
+
+- [#2048](https://github.com/rails-api/active_model_serializers/pull/2048)
+  The output of `render json: MyModel.new` has changed when no serializer
+  is found for `MyModel`. Now, we just call `as_json` on anything that doesn't
+  have a serializer.
```

Comment:
Calling `as_json` when no serializer is found isn't new.  What's new, is that you've describe a regression/change in `render json: MyModel.new`